pkg/actuators/machine/instances.go

```diff
@@ -342,6 +351,24 @@ func launchInstance(machine *machinev1.Machine, machineProviderConfig *awsprovid
 		}
 	}
 
+	instanceTenancy := machineProviderConfig.Tenancy
+
+	if instanceTenancy != "" {
+		if instanceTenancy != defaultTenancy && instanceTenancy != dedicatedTenancy && instanceTenancy != hostTenancy {
+			return nil, mapierrors.CreateMachine("invalid instance tenancy: %s. Allowed options are: %s,%s,%s",
+				instanceTenancy,
+				defaultTenancy,
+				dedicatedTenancy,
+				hostTenancy)
+		}
+
+		if placement == nil {
+			placement = &ec2.Placement{}
+		}
+
+		placement.Tenancy = &machineProviderConfig.Tenancy
+	}
```

Comment:
Would this be simpler as a switch?

```suggestion
	switch instanceTenancy {
		case "":
			// Do nothing when not set
		case defaultTenancy, dedicatedTenancy, hostTenancy:
			if placement == nil {
				placement = &ec2.Placement{}
			}
			placement.Tenancy = &machineProviderConfig.Tenancy
		default:
			return nil, mapierrors.CreateMachine("invalid instance tenancy: %s. Allowed options are: %s,%s,%s",
				instanceTenancy,
				defaultTenancy,
				dedicatedTenancy,
				hostTenancy)
	}
```